composer.json

```diff
@@ -8,18 +8,18 @@
     ],
     "bin": ["bin/svc"],
     "require": {
-        "php": "~7.2.29||~7.3.0||~7.4.0",
+        "php": "^7.3||^8.0",
         "ext-json": "*",
-        "phpstan/phpdoc-parser": "^0.3.5",
+        "laminas/laminas-stdlib": "^3.3",
+        "nikic/php-parser": "~4.4.0||~4.5.0||~4.6.0||~4.7.0",
+        "phpstan/phpdoc-parser": "~0.4.9",
+        "sabre/xml": "^2.1",
         "symfony/console": "~4.1.0||~4.4.0",
         "tomzx/php-semver-checker": "^0.14.0",
-        "wikimedia/less.php": "~1.8.0",
-        "zendframework/zend-stdlib": "^3.2.1",
-        "nikic/php-parser": "~4.4.0||~4.5.0||~4.6.0",
-        "sabre/xml": "^2.1"
+        "wikimedia/less.php": "~1.8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.5.0",
+        "phpunit/phpunit": "9.3.0",
```

Comment:
Can we allow installing 9.3.x?

Reply:
Hello @ihor-sviziev.
I changed it temporary to have some checks.
Now, I fixed all changes and squashed commits.